packages/tools/test-runner/src/index.ts

```
@@ -0,0 +1,585 @@
+      estimatedDuration: file.estimatedDuration
+    }));
+    
+    let shards;
+    
+    switch (this.config.shardStrategy) {
+      case 'duration':
+        // Use historical duration data
+        const patterns = await this.patternRecognizer.detectPatterns(
+          testFiles.map(f => ({ timestamp: Date.now(), value: f.estimatedDuration || 1000 })),
+          { type: 'trend' }
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)